src/code/views/image-search-dialog-view.coffee

```diff
@@ -0,0 +1,140 @@
+
+
+OpenClipart = require '../utils/open-clipart'
+tr = require '../utils/translate'
+
+{div, input, button, img, i, a, form, br} = React.DOM
+
+ImageSearchResult = React.createFactory React.createClass
+  displayName: 'ImageSearchResult'
+  getInitialState: ->
+    loaded: false
+  componentDidMount: ->
+    image = new Image()
+    image.src = @props.imageInfo.image
+    image.onload = =>
+      @setState loaded: true
+  clicked: ->
+    @props.clicked @props.imageInfo
+  render: ->
+    src = if @state.loaded then @props.imageInfo.image else 'img/bb-chrome/spin.svg'
+    if @props.disabled @props.imageInfo
```

Comment:
At first I thought this was a typo. I was wondering how  `if @props.disabled @props.imageInfo` would evaluate.  

Can you write this so its clearer? It seems like `@props.disabled` is a function and `@props.imageInfo` is the argument, but it doesn't immediately read that way.